### PR TITLE
refactor: remove upper cases in import package names #908

### PR DESCRIPTION
**Pull request by aks3213**
# Description

This PR comprises changes related to updating import from camelcase to lowercase and improving package alias name of imports

Closes: [#908](https://github.com/zeta-chain/node/issues/908)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Refactoring code 

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [x] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

**Comment by lumtis** [comment]
@aks3213 thank you so much for the PR, I will have a look today

File: testutil/keeper/fungible.go [hunk 1]
```diff
@@ -43,8 +43,8 @@ func FungibleKeeper(t testing.TB) (*keeper.Keeper, sdk.Context) {
 
 	bankkeeper := bankkeeper2.BaseKeeper{}
 	authkeeper := authkeeper2.AccountKeeper{}
-	evmKeeper := &evmkeeper.Keeper{}
```
Comment (lumtis) [inline testutil/keeper/fungible.go:46]: Why this change?

File: zetaclient/keys_test.go [hunk 2]
```diff
@@ -13,7 +13,7 @@ import (
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	hd "github.com/cosmos/cosmos-sdk/crypto/hd"
-	cKeys "github.com/cosmos/cosmos-sdk/crypto/keyring"
+	cryptokeyring "github.com/cosmos/cosmos-sdk/crypto/keyring"
```
Comment (lumtis) [inline zetaclient/keys_test.go:16]: Or simply `keyring` would also work

**Review by lumtis — COMMENTED** [review]
Great work! Just one comment otherwise looks good.

File: testutil/keeper/fungible.go (hunk 1 above)
Comment (aks3213) [reply testutil/keeper/fungible.go:46]: This seems to be a mistake. Will update it.

File: zetaclient/keys_test.go (hunk 2 above)
Comment (aks3213) [reply zetaclient/keys_test.go:16]: At line 11 in this file its already there "github.com/99designs/keyring".

File: testutil/keeper/fungible.go (hunk 1 above)
Comment (aks3213) [reply testutil/keeper/fungible.go:46]: @lumtis addressed this one. please review.

**Comment by aks3213** [comment]
Let me know if any actions are required from my side..

**Comment by lumtis** [comment]
> Let me know if any actions are required from my side..

You can merge `develop` once https://github.com/zeta-chain/node/pull/936 is merged
